### PR TITLE
Feature/liayoo/tta

### PR DESCRIPTION
**Pull request by liayoo**
DONE
- Added blockchainConfig to node status
- Added /blockchain_config client API
- Added validators, globalTimeSyncStatus to consensus status

TODOs
- Add numberToBlockInfo mapping in BlockPool & /get_block_info_by_number client API
- Extend tools/transfer/sendTransferTx.js
- Update deploy_on_prem.sh (on bitbucket)

### CODE REVIEW

File: consensus/index.js [hunk 1]
```diff
@@ -1405,13 +1411,23 @@ class Consensus {
...
+   *   stateNumeric,
+   *   epoch,
+   *   validators,
+   *   globalTimeSyncStatus
    * }
    */
   getStatus() {
     const lastFinalizedBlock = this.node.bc.lastBlock();
+    const validators = this.validators;
+    const globalTimeSyncStatus = this.ntpData;
```
Comment (platfowner) [inline consensus/index.js:1425]: question: does this.ntpData include when the sync was done? Unless the answer is yes, would it helpful to add when the data was fetched with field name like 'syncedAt'?

Comment (liayoo) [reply consensus/index.js:1425]: Great idea! 👍 Done!

**Comment by liayoo** [comment]
Addressed all comments, PTAL @platfowner

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Pull request merged**